README.md

```diff
@@ -364,6 +364,22 @@ It **MAY** be a requirement and the server can send back `428 Precondition Requi
 This **MAY** be used for GET requests e.g don’t download data that hasn’t changed since the client last requested it
 
 
+###Remove properties of a resource
+Removes properties of a single resource using the PATCH verb and a subset (remove operation) of [JSON PATCH](https://tools.ietf.org/html/rfc6902) semantics.
+
+Content-Type must be set to application/json-patch+json
+
+*Example request*
+
+    PATCH /datasets/drafts/{id}
+    [
+     { "op": "remove", "path": "/name" }
```

Comment:
Yep - that's cool - I just didn't want everything to be implemented but only 'remove' documented.